src/model/bpmn/shape/ShapeBpmnElement.ts

```diff
@@ -20,12 +20,24 @@ export default class ShapeBpmnElement {
   constructor(readonly id: string, readonly name: string, readonly kind: ShapeBpmnElementKind, public parentId?: string, readonly instantiate: boolean = false) {}
 }
 
+export type BpmnEventKind =
```

Comment:
I don't know what name uses. It can be confusing with ShapeBpmnEventKind.

If you are a bettre idea, I'm ok :)

Reply:
Decisions: keep the name and move to ShapeUtil. Renaming will be reviewed through #158